code/modules/power/antimatter/control.dm

```diff
@@ -47,7 +52,14 @@
 
 /obj/machinery/power/am_control_unit/process()
 	if(exploding)
-		explosion(get_turf(src),8,12,18,12)
+		radio.talk_into(src, "REACTOR MELTDOWN IMMINENT. EVACUATE SURROUNDING AREA.", common_channel)
+		radiation_pulse(src, 3000, 50, TRUE)
+		playsound(src, 'sound/effects/neovgre_exploding.ogg', 100)
+		explosion(get_turf(src),30,40,50,60)
+		for(var/mob/M in GLOB.player_list)
+			if(!isnewplayer(M) && M.can_hear())
+				SEND_SOUND(M, 'sound/effects/explosion3.ogg') //everyone goan know bout this
```

Comment:
```suggestion
				SEND_SOUND(M, 'sound/f13effects/explosion_distant_4.ogg') //everyone goan know bout this
```
This sounds better for something that explodes miles away AND underground, IMHO.